app.go

```diff
@@ -765,15 +755,15 @@ func (app *App) start(ctx context.Context) error {
 	// Attempt to start cleanly.
 	if err := app.lifecycle.Start(ctx); err != nil {
 		// Start failed, rolling back.
-		fxlog.Info("startup failed, rolling back", fxlog.Err(err)).Write(app.log)
+		app.log.LogEvent(&fxlog.StartErrorEvent{Err: err})
```

Comment:
Oh, I see that we have a StartErrorEvent here. We could name this RollbackEvent, since that's what we're doing next. (We're usually emitting events right we do a thing, unless it can only be done after -- like receiving an error.)

If we call this RollbackEvent, the Err parameter can be StartErr: the error that caused the rollback to happen.